source/NameDialog.cpp

```diff
@@ -0,0 +1,52 @@
+/* NameDialog.cpp
+Copyright (c) 2024 by Endless Sky contributors
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "NameDialog.h"
+
+#include "Color.h"
+#include "text/Font.h"
+#include "text/FontSet.h"
+#include "SpriteSet.h"
+#include "SpriteShader.h"
+using namespace std;
+
+
+
+void NameDialog::Draw()
+{
+	Dialog::Draw();
+
+	randomPos = cancelPos - Point(100., 0.);
+	SpriteShader::Draw(SpriteSet::Get("ui/dialog cancel"), randomPos);
+
+	const Font &font = FontSet::Get(14);
+	static const string RANDOM = "Suggest";
```

Comment:
```suggestion
	static const string RANDOM = "Random";
```

This is the name for the same button when purchasing a new ship
![image](https://github.com/endless-sky/endless-sky/assets/109186806/31673484-c212-4ce6-82bb-d9dce32ef420)

Reply:
The point of the PR is to change the name of this button + add it to the other rename dialog.

Reply:
I'm in favor of adding this button to other locations, but I don't think it should be renamed.